### PR TITLE
Open webcam panel action

### PR DESCRIPTION
**Pull request by malloxpb**
+ Add webcam panel to the Webcam Panel
+ Open the webcam in the new window, turn off the webcam when the window is closed

### CODE REVIEW

File: src/main/java/org/vietabroader/view/MainView.java [hunk 1]
```diff
@@ -258,8 +264,38 @@ private JPanel createWebcamPanel() {
...
         c.anchor = GridBagConstraints.CENTER;
-        panel.add(new JButton("Start Webcam"), c);
+        JButton webcamButton = new JButton("Start Webcam");
+        panel.add(webcamButton, c);
+        webcamButton.addActionListener(new MyAction());
 
         return panel;
     }
+
+    public class MyAction implements ActionListener {
```
Comment (ctring) [inline src/main/java/org/vietabroader/view/MainView.java]: Rename to something more descriptive. OpenWebcamPanelAction for example

File: src/main/java/org/vietabroader/view/MainView.java [hunk 2]
```diff
@@ -258,8 +264,38 @@ private JPanel createWebcamPanel() {
...
+        private Webcam webcam = null;
+        private WebcamPanel panel = null;
+
+        public void actionPerformed(ActionEvent ae){
+            JFrame frame2 = new JFrame("QR code");
```
Comment (ctring) [inline src/main/java/org/vietabroader/view/MainView.java]: Rename to webcamFrame

**Review by ctring — CHANGES_REQUESTED** [review]
Very good job!

**Comment by malloxpb** [comment]
I have changed the names for classes and var as requested :)

**Comment by ctring** [comment]
branch

I'm not sure what happened but your branch look really weird. First is that the commits are duplicated and second is that you have 2 branches merge into one. If we follow our workflow correctly, which only involves rebasing, our branch will always have one line. If you have two branch merging into one then you're not rebasing. In the end, the code base is still the same but the history when doing rebase only is much cleaner.

**Comment by malloxpb** [comment]
yes I see that. The problem is that I ran rebase after I committed some changes to the branch. I will pay attention to that closer next time :)

**Pull request merged**